dace/transformation/transformation.py

```diff
@@ -446,13 +586,29 @@ def apply(self, sdfg: SDFG):
     @classmethod
     def apply_to(cls,
                  sdfg: SDFG,
-                 *where: Union[nd.Node, SDFGState, SubgraphView],
+                 *where: Union[nd.Node, SDFGState, gr.SubgraphView],
```

Comment:
SDFGState without module name

Reply:
Same as above, I won't be able to fix all existing cases here.